VideoApi/VideoApi.DAL/Commands/UpdateRoomCommand.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Exceptions;
+
+namespace VideoApi.DAL.Commands
+{
+    public class UpdateRoomCommand : ICommand
+    {
+        public UpdateRoomCommand(Guid conferenceId, long roomId, string label, string ingestUrl, string pexipNode,
```

Comment:
UpdateRoomConnectionDetailsCommand

Reply:
good spot. updated the class name and the domain method to be `update` rather than `add`